sample acclerometer(source yt)/MainActivity.java

```diff
@@ -0,0 +1,68 @@
+package com.example.myapplicationas;
+
+import androidx.appcompat.app.AppCompatActivity;
+import android.hardware.Sensor;
+import android.hardware.SensorEvent;
+import android.hardware.SensorEventListener;
+import android.hardware.SensorManager;
+import android.os.Bundle;
+import android.widget.TextView;
+/*CODE FROM
+* YT-Android Tutorial Spot
+* video link-https://youtu.be/LsWJipo4knk*/
+public class MainActivity extends AppCompatActivity implements SensorEventListener {
+/*only the abv lines i understand*/
+
+    private TextView textView;
+    private SensorManager sensorManager;
+    private Sensor sensor;
+
+
+
+
+    /*i dnt know what type of variable or objects are created abv

```

Comment:
I am not familiar either with what a Sensor class does in this case. Is not important that you know what it is, you just need to know ‘ok sensor is an object of type Sensor, I guess it will have some method like readSensorData or something similar. For now i will just keep in mind we have this class Sensor, let’s see how it is used in the code below’

Reply:
If you really want to get to know that just ‘dig in the code’ so control click in vs code and check where this Sensor class is defined and try to see what methods and attributes it has to get an idea of what it does. But since right now we are reading code is not necessary, you want to dig in the code when you are r writing your own code and you are looking for a class that does something that you want